piton/tools/src/sims/metro_chipset.config

```diff
@@ -138,11 +138,30 @@
     -vcs_build_args=+nospecify
     //-vcs_build_args=+nbaopt
     -vcs_build_args=-Xstrict=1 -notice
-    -vcs_build_args=-P $DV_ROOT/tools/pli/iop/bwioj.tab // needed for fake_l2
-    -vcs_build_args=-P $DV_ROOT/tools/pli/socket/bwsocket_pli.tab
-    -vcs_build_args=-P $DV_ROOT/tools/pli/mem/bwmem_pli.tab
-    -vcs_build_args=-lsocket_pli -liob -lmem_pli
+    //-vcs_build_args=-P $DV_ROOT/tools/pli/iop/bwioj.tab // needed for fake_l2
+    //-vcs_build_args=-P $DV_ROOT/tools/pli/socket/bwsocket_pli.tab
+    //-vcs_build_args=-P $DV_ROOT/tools/pli/mem/bwmem_pli.tab
+    //-vcs_build_args=-lsocket_pli -liob -lmem_pli
     -vcs_build_args=+rad
+    -vcs_build_args="-LDFLAGS -Wl,--no-as-needed"
+    -config_rtl=PITON_DPI 
+    -vcs_build_args="-CFLAGS -DPITON_DPI" 
+    -vcs_build_args="-CFLAGS -I/home/brianli/utils/openmpi/include" 
```

Comment:
What's the deal with these includes and linker arguments? Do we need them all? Why doesn't mpic++ just know where they are?

Reply:
VCS only allows for user-defined compilers like mpic++ in some places and overrides it with g++ in some parts of the Makefile it produces resulting in errors, so the workaround is to grab the output of `mpic++ --showme` and use them as CFLAGS. `mpic++ --showme` gives you the code that the arguments the wrapper is calling on top of g++, which is why it is currently hardcoded to my directories. We could specify to users to do this step themselves?

Reply:
could you try running it without these and sending me the log on slack/zulip? I'd like to see what the errors are

Reply:
ok we resolved this with correctly setting `-cc mpicc` and `-cpp mpic++`